### ISSUE TITLE
One selector per line.

### ISSUE THREAD

**Issue by manix84** [CSSLint/csslint#280]
# Can you add an option to force a single selector per line.

I've always found it a safe rule to follow.

This would fail:

``` css
.selector1 div span a, .selector2 div span a, .selector3 div span a {
    color: #fff;
}
```

This would pass:

``` css
.selector1 div span a,
.selector2 div span a,
.selector3 div span a {
    color: #fff;
}
```
